code/script.js

```diff
@@ -1,7 +1,14 @@
 // All the DOM selectors stored as short variables
-const board = document.getElementById('board')
-const questions = document.getElementById('questions')
-const restartButton = document.getElementById('restart')
+const board = document.getElementById('board');
+const questions = document.getElementById('questions');
+const restartButton = document.getElementById('restart');
+const findOutButton = document.getElementById('filter');
+const winOrLoseText = document.getElementById('winOrLoseText');
+const winOrLoseSection = document.getElementById('winOrLose');
+const playAgain = document.getElementById('playAgain');
+const numberOfguesses = document.getElementById('numberOfGuesses');
+
```

Comment:
Nice structure in the beginning and the end of the code with the documents on top and the eventlisterners at the bottom.